HACKING.md

```diff
@@ -26,12 +26,14 @@ Once a release has been created, it can be pushed:
 To cut an official tag release, we generally use the images built by [ci.openshift.redhat.com](https://ci.openshift.redhat.com)
 under the openshift3_ami job.
 
+1. Make sure the "openshift3_ami" job is passing
+1. Pause the merge queue by disabling the "process_pull_requests" job on [ci.openshift.redhat.com](https://ci.openshift.redhat.com)
 1. Create a new git tag `git tag v0.X.X -a -m "v0.X.X" HEAD`
-2. Push the tag to GitHub `git push origin --tags` where `origin` is `github.com/openshift/origin.git`
-3. Run the "openshift3_ami" job
-4. Once the images are pushed to the repository, run `OS_PUSH_TAG="v0.X.X" hack/push-release.sh`. Your tag must match the Git tag.
-5. Upload the binary artifacts generated by that build to GitHub release page
-6. Send an email to the dev list, including the important changes prior to the release.
+1. Tag the images that previously passed in the "openshift3_ami" build by running `OS_PUSH_TAG="v0.X.X" hack/push-release.sh`. Your tag must match the Git tag.
```

Comment:
After I rebuilt the "openshift3_ami" job I would then tag the "openshift/origin" image to the correct version.  I'm guessing that's not the way you do it.